mapUtils.js

```diff
@@ -0,0 +1,54 @@
+const sm = new (require("@mapbox/sphericalmercator"))({ size: 512 });
+
+const overlayLineLayerDef = {
```

Comment:
I wonder if these overlay layers should be in their own file or somewhere that makes it easier to configure 🤔